server/src/main/java/org/elasticsearch/common/logging/Loggers.java

```diff
@@ -71,29 +67,19 @@ public static Logger getLogger(Class<?> clazz, Settings settings, Index index, S
     }
 
     public static Logger getLogger(Class<?> clazz, Settings settings, String... prefixes) {
-        final List<String> prefixesList = prefixesList(settings, prefixes);
-        return Loggers.getLogger(clazz, prefixesList.toArray(new String[prefixesList.size()]));
+        return Loggers.getLogger(clazz, prefixes);
     }
 
     public static Logger getLogger(String loggerName, Settings settings, String... prefixes) {
```

Comment:
These `Settings` variants are no longer needed and I propose to remove them in a follow up PR.